dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLVectorStoreRecordCollection.cs

```diff
@@ -659,5 +687,32 @@ private async IAsyncEnumerable<JsonObject> GetItemsAsync(QueryDefinition queryDe
         }
     }
 
+    /// <summary>
+    /// Get vector property to use for a search by using the storage name for the field name from options
+    /// if available, and falling back to the first vector property in <typeparamref name="TRecord"/> if not.
+    /// </summary>
+    /// <param name="optionsVectorFieldName">The vector field name provided via options.</param>
+    /// <exception cref="InvalidOperationException">Thrown if the provided field name is not a valid field name.</exception>
+    private VectorStoreRecordVectorProperty? GetVectorPropertyForSearch(string? optionsVectorFieldName)
```

Comment:
nit. (Privates) 

Not sure why this is called `options` while there's just one option. 

In the error you are referring as `Vector Property Name` but the argument here is using a different naming, for consistency I would use one or another.

```suggestion
    private VectorStoreRecordVectorProperty? GetVectorPropertyForSearch(string? vectorPropertyName)
```

Reply:
> Not sure why this is called options while there's just one option.

`options` part means that this vector field name came from `VectorSearchOptions` class, but I agree that from method perspective it doesn't matter from where this property came from. Updated.